code/modules/client/preference_setup/loadout/loadout_factions.dm

```diff
@@ -1,28 +1,20 @@
 /datum/gear/faction
 	display_name = "idris cap"
-	path = /obj/item/clothing/head/softcap/security/idris
+	path = /obj/item/clothing/head/softcap/idris
 	slot = slot_head
 	sort_category = "Factions"
 	cost = 1
 	faction = "Idris Incorporated"
 
 /datum/gear/faction/idris_beret
 	display_name = "idris beret"
-	path = /obj/item/clothing/head/beret/security/idris
+	path = /obj/item/clothing/head/beret/corporate/idris
 	faction = "Idris Incorporated"
 
 /datum/gear/faction/idris_beret_alt
 	display_name = "idris beret (alt)"
-	path = /obj/item/clothing/head/beret/security/idris/alt
+	path = /obj/item/clothing/head/beret/corporate/idris/alt
 	faction = "Idris Incorporated"
-
-/datum/gear/faction/idris_uniform_alt
-	display_name = "idris service skirt"
```

Comment:
Why are we removing this? It's an alternative to the same uniform except it's a skirt.

Reply:
actually, the idris uniforms have been changed a lot, so much so that this only really works as an alternative to the chef outfit. which is why i removed it.

once alternative jobspawn outfits start working for factions i'll be able to add alt outfits proper